packages/components/tests/dummy/app/templates/components/app-footer.hbs

```diff
@@ -0,0 +1,396 @@
+            <AF.ExtraAfter>
+              <Shw::Placeholder @height="2em" {{style width="fit-content"}} @text="After" @background="#FF7575" />
+            </AF.ExtraAfter>
+          </Hds::AppFooter>
+        </div>
+      </SF.Item>
+
+      <SF.Item @label="Mobile (large) view">
+        <div class="shw-component-app-footer-mobile-large-view">
+          <Hds::AppFooter as |AF|>
+            <AF.ExtraBefore>
+              <Shw::Placeholder @height="2em" {{style width="fit-content"}} @text="Before" @background="#FF7575" />
+            </AF.ExtraBefore>
+            <AF.StatusLink @status="operational" />
+            <AF.LegalLinks />
+            <AF.Item>
+              <Hds::Text::Body @tag="span" @size="100">v.2.0</Hds::Text::Body>
+            </AF.Item>
+            <AF.Item>
+              <Hds::Text::Body @tag="span" @size="100">Name</Hds::Text::Body>
+            </AF.Item>
+            <AF.Item>
+              <Hds::Text::Body @tag="span" @size="100">API: 1.0</Hds::Text::Body>
+            </AF.Item>
+            <AF.ExtraAfter>
+              <Shw::Placeholder @height="2em" {{style width="fit-content"}} @text="After" @background="#FF7575" />
+            </AF.ExtraAfter>
+          </Hds::AppFooter>
+        </div>
+      </SF.Item>
+
+      <SF.Item @label="Mobile (small) view">
+        <div class="shw-component-app-footer-mobile-small-view">
+          <Hds::AppFooter as |AF|>
+            <AF.ExtraBefore>
+              <Shw::Placeholder @height="2em" {{style width="fit-content"}} @text="Before" @background="#FF7575" />
+            </AF.ExtraBefore>
+            <AF.StatusLink @status="operational" />
+            <AF.LegalLinks />
+            <AF.Item>
+              <Hds::Text::Body @tag="span" @size="100">v.2.0</Hds::Text::Body>
+            </AF.Item>
+            <AF.Item>
+              <Hds::Text::Body @tag="span" @size="100">Name</Hds::Text::Body>
+            </AF.Item>
+            <AF.Item>
+              <Hds::Text::Body @tag="span" @size="100">API: 1.0</Hds::Text::Body>
+            </AF.Item>
+            <AF.ExtraAfter>
+              <Shw::Placeholder @height="2em" {{style width="fit-content"}} @text="After" @background="#FF7575" />
+            </AF.ExtraAfter>
+          </Hds::AppFooter>
+        </div>
+      </SF.Item>
+    </Shw::Flex>
+
+    <Shw::Divider />
+
+    <Shw::Text::H2>Base components & child components</Shw::Text::H2>
+
+    <button type="button" {{on "click" this.toggleHighlight}}>
+      {{if this.showHighlight "Hide" "Show"}}
+      layout highlight
+    </button>
+
+    <Shw::Text::H3>Hds::AppFooter::Item</Shw::Text::H3>
+
+    <ul class="hds-app-footer__list">
+      <Hds::AppFooter::Item><Hds::Text::Body
+          @tag="span"
+          @size="100"
+          @color="primary"
+        >Item</Hds::Text::Body></Hds::AppFooter::Item>
+    </ul>
+
+    <Shw::Divider @level={{2}} />
+
+    <Shw::Text::H3>Hds::AppFooter::Link</Shw::Text::H3>
+
+    <Shw::Text::H4>States</Shw::Text::H4>
```

Comment:
I think it's clearer to label the themes so you know what you are looking at. Going to leave as is for now.

Reply:
Pretty self evident in this case, for me (also because they're described right above). If you feel strongly on this, revert my last commit that changed the showcase (sorry, I read this comment only now)